### DISCUSSION TITLE
Disable extension modification or hide it

### DISCUSSION THREAD

**Discussion by voodoomsr** [mozilla/policy-templates#843] [Ideas]
Hi everyone, I would like to know if it is possible to block modifying an extension or if that isn't possible to hide it.
I already found the mechanism to lock its removal, but still I can interact with the extension trough its menus.
```{
  "policies": {
    "Extensions": {
      "Locked": ["myallychou@gmail.com"]
    }
  }
}
```

Why this could be useful? well there are extensions for example that block certain content. In the example above the extension is unhook that hides the recommendation part of YouTube. 
I would like to enforce that extension behavior so it can't be disabled so easily. The extension itself has a turn on/off switch, so even though is locked I can turn it off in 2 clicks.

**Comment by mkaply** [comment]
So I'm looking into hiding the button, but the user could probably still get to the settings somewhere else. 

The best solution is to ask the extension provider to provide group policy for their extension and then you could lock it.
